### PR TITLE
Add EngCom Slack info to README and formatting

### PR DESCRIPTION
**Pull request by lorikrell**
### Description (*)
No code changes
Add Community Engineering SLACK info to README
Minor formatting updates to page

### Fixed Issues (if relevant)
No issue associated
EngCom request

### Manual testing scenarios (*)
No testing need, no code change

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=19696) <br/>All committers have signed the CLA.

**Comment by magento-engcom-team** [comment]
Hi @lorikrell. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

File: README.md [hunk 1]
```diff
@@ -59,3 +59,13 @@ Please see [LICENSE.txt](https://github.com/magento/magento2/blob/2.3-develop/LI
 Subject to Licensee's payment of fees and compliance with the terms and conditions of the MEE License, the MEE License supersedes the OSL 3.0 license for each source file.
 Please see LICENSE_EE.txt for the full text of the MEE License or visit https://magento.com/legal/terms/enterprise.
 
+## Community Engineering Slack
+
+To connect with Magento and the Community, join us on the [Magento Community Engineering Slack](https://magentocommeng.slack.com). If you are interested in joining Slack, or a specific channel, send us request at [engcom@magento.com](mailto:engcom@magento.com) or [self signup](https://tinyurl.com/engcom-slack).
```
Comment (okorshenko) [inline README.md]: please use engcom@adobe.com

Comment (lorikrell) [reply README.md]: Updated in latest commit.

**Review by okorshenko — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @okorshenko, thank you for the review.
[ENGCOM-3662](https://jira.corp.magento.com/browse/ENGCOM-3662) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
Hi @lorikrell. Thank you for your contribution. 
We will aim to release these changes as part of 2.3.1. 
Please check the release notes for final confirmation.

**Pull request merged**